.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,88 @@
+name: Bug Report
+description: File a bug report
+title: "[BUG]: "
```

We already have the labels, I'd skip the prefix:

```suggestion
title: ""
```